.github/workflows/integration.yml

```diff
@@ -5,11 +5,15 @@ name: Notebooks for Integration Tests
 
 on:
   pull_request:
-    types: [ labeled ]
+      paths:
```

Comment:
I believe if we keep here as:
```
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```
Then keeping the condition `if: ${{ github.event.label.name == 'integration' }}` should be sufficient to trigger the test on every push, if the event label is 'integration`.

Reply:
I am not 100% sure, though. Therefore, it worths testing.